CMakeLists.txt

```diff
@@ -1,13 +1,19 @@
 cmake_minimum_required(VERSION 3.10)
-project(vikunja)
+project(vikunja
+  VERSION      0.1.0
+  DESCRIPTION  "Primitives for Alpaka"
+  HOMEPAGE_URL "https://github.com/ComputationalRadiationPhysics/vikunja"
+  LANGUAGES    CXX)
 
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
 option(VIKUNJA_REDUCE_COMPARING_BENCHMARKS_ENABLE "Should the reduce integration test run some comparison benchmarks?" OFF)
 option(VIKUNJA_TRANSFORM_COMPARING_BENCHMARKS_ENABLE "Should the transform integration test run some comparison benchmarks?" OFF)
 
-list(APPEND CMAKE_MODULE_PATH "${ALPAKA_ROOT}")
-set(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED ON)
+if(${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME})
+  list(APPEND CMAKE_MODULE_PATH "${ALPAKA_ROOT}")
+  option(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE "enable alpaka serial accelerator" ON)
+endif()
 find_package(alpaka REQUIRED)
```

Comment:
You can also specify a version here if required. This way the build system can detect incompatible versions if vikunja is used together with cupla.

```cmake
find_package(alpaka [version] [EXACT] REQUIRED)
```